src/xrc/src/request_log.rs

```diff
@@ -0,0 +1,59 @@
+use std::{cell::RefCell, collections::VecDeque, thread::LocalKey};
+
+use candid::Principal;
+use ic_xrc_types::{GetExchangeRateRequest, GetExchangeRateResult};
+
+pub(crate) struct RequestLogEntry {
```

Comment:
Could you please add some documentation to this file as well?

Reply:
Added!